src/web/aws/authentication/AWSAuthenticationTypes.jsx

```diff
@@ -63,32 +69,47 @@ const AWSAuthenticationTypes = ({ onChange }) => {
     }
   };
 
+  let authTypes;
```

Comment:
Instead of creating a let (we usually try to avoid those) we can move this logic into the return directly:


```suggestion
return(
  <>
    {AppConfig.isCloud() ? 
      <KeySecret
        awsKey={awsCloudWatchAwsKey}
        awsSecret={awsCloudWatchAwsSecret}
        onChange={onChange}
      /> : (
      <>
        <Input type="select"
               name="awsAuthType"
               id="awsAuthType"
               onChange={handleTypeChange}
               label="AWS Authentication Type"
               defaultValue={currentType}>
          {Object.keys(AWS_AUTH_TYPES).map((type) => (
            <option value={AWS_AUTH_TYPES[type]}
                    key={`option-${type}`}>
              {AWS_AUTH_TYPES[type]}
            </option>
          ))}
        </Input>

        <AuthWrapper>
          {isType(AWS_AUTH_TYPES.automatic) && <Automatic />}

          {isType(AWS_AUTH_TYPES.keysecret) && (
            <KeySecret awsKey={awsCloudWatchAwsKey}
                       awsSecret={awsCloudWatchAwsSecret}
                       onChange={onChange} />
          )}
        </AuthWrapper>
      </>
    )
    <ARN awsARN={awsCloudWatchAssumeARN} onChange={onChange} />
  </>
);
```